libs/island-ui/core/src/lib/Box/types.ts

```diff
@@ -5,4 +5,5 @@ export interface BoxProps
   extends Omit<UseBoxStylesProps, 'component'>,
     Omit<AllHTMLAttributes<HTMLElement>, 'width' | 'height' | 'className'> {
   component?: ElementType
+  dataTestId?: string
```

Comment:
Not sure if it makes sense, but could we maybe use [this type](https://github.com/island-is/island.is/blob/main/libs/island-ui/utils/src/lib/testSupport.ts) instead of specifying this explicitly?

Reply:
I thought we'd removed it, since I couldn't find it. Using it now.